File: problems/92/Main.cpp
#include <iostream>
#include <cstdio>
#include <algorithm>
#include <cstring>
#include <string>
#include <cctype>
#include <stack>
#include <queue>
#include <list>
#include <vector>
#include <map>
#include <sstream>
#include <cmath>
#include <bitset>
#include <utility>
#include <set>
#include <numeric>
#include <ctime>
#include <math.h>

using namespace std;

class Solution {
	public:
		vector<vector<int> > generateMatrix(int A);
};

//--SPLIT--

//-vector<vector<int> > Solution::generateMatrix(int A) {
//-}

//--SPLIT--
vector<vector<int> > Solution::generateMatrix(int A) {
	vector<vector<int> > result(A);
	return result;
}

//--SPLIT--
int main() {
	int testcases;
	cin >> testcases;
	for (int t = 0; t < testcases; t++) {
		int A; cin >> A;
		Solution solution;
		vector<vector<int> > result = solution.generateMatrix(A);
		for (int j = 0; j < result.size(); j++) {
			vector<int> v = result[j];
			for (int k = 0; k < v.size(); k++) {
				cout << v[k]  << " ";
			}
			cout << "\n";
		}
	}
	return 0;
}

File: problems/92/Main.py
from __future__ import print_function
import math


#--SPLIT--
#-class Solution:
	# @param A : integer
	# @return a list of list of integers
#-	def generateMatrix(self, A):




#--SPLIT--
class Solution:
	# @param A : integer
	# @return a list of list of integers
	def generateMatrix(self, A):
		return []



#--SPLIT--

if __name__ == "__main__":
	test_cases = int(raw_input())
	for i in range(test_cases):
		A = int(raw_input())
		solution = Solution()
		result = solution.generateMatrix(A)
		for r in result:
			for x in r:
				print(str(x) + " ", end="")
			print("")
